src/mpc/cli/mpc_phase2_contribute.cpp

```diff
@@ -0,0 +1,158 @@
+#include "mpc_common.hpp"
+#include "snarks/groth16/mpc/phase2.hpp"
+
+using namespace libzeth;
+namespace po = boost::program_options;
+
+namespace
+{
+
+// Usage:
+//   $0 phase2-contribute [<options>] <challenge_file>
+//
+// Options:
+//   --out <file>        Response output file (mpc-response.bin)
```

Comment:
Is `(mpc-response.bin)` the default filename?

Reply:
Yes, exactly.  I think this is a fairly common way to express it.  Let me know if you prefer some other way.

Reply:
I'd write something like: `--out <file>        Response output file (default "mpc-response.bin")`

In addition to that; I'd be in favor to use environment variables to avoid creating files in the location where the command is ran. Maybe we can use the env var `ZETH_TRUSTED_SETUP_DIR` for that, or just $HOME

Reply:
Based on our discussion, I've removed these --out flags from all the mpc subcommands, so the user is forced to specify the output file explicitly.